terraform/cleanup/cleanup.sh

```diff
@@ -97,13 +100,34 @@ cleanup()
 	if test "${3:0:2}" == "--"; then cleanup_list "$1" "" "" "$RL"; else cleanup_list "$1" "" "$3" "$RL"; fi
 }
 
+# Find volumes attached to a set of servers
+# $* => list of server UUIDs
+# Output a list of volume UUIDs that are attached
+server_vols()
+{
+	if test -z "$1"; then echo; return 1; fi
+	srchexpr="\\("
+	for arg in "$@"; do
+		srchexpr="$srchexpr$arg\\|"
+	done
+	srchexpr="${srchexpr%\\|}\\)"
+	#if test -n "$DEBUG"; then echo "#### Search for volumes $srchexpr"; fi
```

Comment:
Should this be commented out?

Reply:
It is commented out already ... so I may fail to understand your suggestion?

Reply:
I meant it more like if it is a mistake that it is commented. But does not seem like it.